test/python/fx_importer/sparse_test.py

```diff
@@ -247,14 +258,17 @@ def forward(self, x, y):
     print(m)
 
     # Run it with PyTorch torch.sparse and with TORCH-MLIR sparse_jit.
-    # TODO: run with COO, right now only CSR works
     sparse_input = dense_input.to_sparse_csr()
```

Comment:
remove this csr() and res3

(that was the original intent of this test, i.e. run it only in the way we CHECK the IR above)